### DISCUSSION TITLE
What's your example images model?

### DISCUSSION THREAD

**Discussion by maniakid** [Extraltodeus/multi-subject-render#20] [General]
HI, @Extraltodeus ,
Thanks for your work, I also like your example images results, want to know what model you're using, anywhere can I find this model?

**Comment by Extraltodeus** [comment]
1: 50/50 f222+sfeb

2: 70/30 f111+previous result

3: 50/50 berry+previous result

4: 80/20 SD1.5+previous result

**Reply by Extraltodeus** [reply]
hassan blend seems to give similar results

**Reply by maniakid** [reply]
Thanks a lot.
